enterprise/backend/src/metabase_enterprise/serialization/v2/storage.clj

```diff
@@ -1,15 +0,0 @@
-(ns metabase-enterprise.serialization.v2.storage
```

Comment:
I didn't expect a PR changing the logs will change the API, is there context for this?

Reply:
It's just an implementation detail. The namespace defined a single multimethod which had one implementation.

Reply:
I think we can cross that bridge when we come to it. There are lots of opportunities for streamlining code without breaking anything, and while making it easy to change in the future. We don't have to decide on a dispatch approach before we have more than one backend. We may find that there is a better approach when we come to it.